pyV2DL3/fillEVENTS.py

```diff
@@ -165,4 +165,13 @@ def fillEVENTS(datasource, save_multiplicity=False, instrument_epoch=None, event
         logging.debug("Keyword QUALITY not set in the EVENTS header")
         logging.debug("For EventdisplayAnalysis: use version >=v486")
 
+    try:
+        hdu1.header.set(
+            "NSBLEVEL",
+            evt_dict["NSBLEVEL"],
+            "Measure for NSB level",
+        )
```

Comment:
Do we also want to add more information about? what it stands for? since we are using `*pedvar` as a variable and header keyword is `NSBLEVEL`.

Reply:
added here [58c1033](https://github.com/VERITAS-Observatory/V2DL3/pull/150/commits/58c10339776ec4691db9b3e7bcb5e02ec978ee0b)